src/core/ext/filters/client_channel/resolver/dns/c_ares/dns_resolver_ares.cc

```diff
@@ -460,10 +460,11 @@ static grpc_address_resolver_vtable ares_resolver = {
     grpc_resolve_address_ares, blocking_resolve_address_ares};
 
 #ifdef GRPC_UV
-/* TODO(murgatroid99): Remove this when we want the cares resolver to be the
- * default when using libuv */
+/* TODO(murgatroid99): Remove this when c-ares support is no longer disabled for
+ * custom IO managers */
 static bool should_use_ares(const char* resolver_env) {
-  return resolver_env != nullptr && gpr_stricmp(resolver_env, "ares") == 0;
+  return resolver_env == nullptr || strlen(resolver_env) == 0 ||
```

Comment:
This is now essentially the same as the version on line 470 (the only difference being the check of `!g_custom_iomgr_enabled`).  Should we just use the same code for libuv as for other platforms now?

Reply:
The problem is that the libuv iomgr is a custom iomgr so it sets that flag. I don't think there's another runtime flag for checking for libuv specifically, so the best solution I see is to use a compile time check to not check that runtime flag. But I want it to otherwise have the same functionality as the other implementation. This is why I changed the "TODO". Once c-ares is enabled for other custom iomgrs, the `!g_custom_iomgr_enabled` check should be removed and then the implementations will be the same again.